iosApp/iosApp/ComponentViews/TransitHeader.swift

```diff
@@ -27,6 +27,7 @@ struct TransitHeader<Content: View>: View {
                 .foregroundStyle(textColor)
                 .textCase(.none)
                 .frame(maxWidth: .infinity, maxHeight: modeIconHeight, alignment: .leading)
+                .contentShape(.rect)
```

Comment:
This might be one of the least intuitive fixes I’ve ever implemented. I’d expect the content shape to already be a rectangle, but apparently setting it to be a rectangle also makes its size cover the full `.frame()` size.